### ISSUE TITLE
Mask discriminator missing (?)

### ISSUE THREAD

**Issue by valecorbetta** [spthermo/SDNet#3]
Hi, 
thanks a lot for the Pytorch implementation of SDNet. I was comparing your code with the original paper and Keras implementation and I was wondering were the mask discriminator for the semi-supervised approach is in your code or if it's missing for some specific reasons. 

Thanks a lot in advance

**Comment by spthermo** [comment]
Hi, thank you for the kind words! It's true that this SDNet variant was implemented with a specific focus to stress out the content stream and its possible alternative architectures in the full supervision scenario, without taking into account the semi-supervised approach. So yes, the mask discriminator (and by extension the corresponding loss function) has been intentionally omitted.
